CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes for each version of this project will be documented in this file.
 
+## 17.1.0
+### General
```

Comment:
Heheh, might've been too diligent transferring changes. Don't think we want the changelog at all since this might not end up in 17.1 directly, plus that's already there. Might also want to drop the schematics stuff, I think we're mostly interested in the commits that modify the actual source only. Will see if that can be done by skipping other commits only